### PR TITLE
Fix C_BlockLegacy hooks (Attempt 2)

### PR DESCRIPTION
**Pull request by brampedgex**
Forgot about some last minute changes I made to the signature, it should be working now.

### CODE REVIEW

File: Memory/Hooks.cpp [hunk 1]
```diff
@@ -42,19 +42,19 @@ void Hooks::Init() {
...
-			uintptr_t** blockLegacyVtable = reinterpret_cast<uintptr_t**>(sigOffset + offset + 7);
+			uintptr_t** blockLegacyVtable = reinterpret_cast<uintptr_t**>(sigOffset + offset + /*length of instruction*/ 7);
 			if (blockLegacyVtable == 0x0 || sigOffset == 0x0)
 				logF("C_BlockLegacy signature not working!!!");
 			else {
-				g_Hooks.BlockLegacy_getRenderLayerHook = std::make_unique<FuncHook>(blockLegacyVtable[125], Hooks::BlockLegacy_getRenderLayer);
+				g_Hooks.BlockLegacy_getRenderLayerHook = std::make_unique<FuncHook>(blockLegacyVtable[166], Hooks::BlockLegacy_getRenderLayer);
 
-				g_Hooks.BlockLegacy_getLightEmissionHook = std::make_unique<FuncHook>(blockLegacyVtable[16], Hooks::BlockLegacy_getLightEmission);
+				g_Hooks.BlockLegacy_getLightEmissionHook = std::make_unique<FuncHook>(blockLegacyVtable[170], Hooks::BlockLegacy_getLightEmission);
```
Comment (GodGamer029) [inline Memory/Hooks.cpp:55]: u sure on this one?

Comment (brampedgex) [reply Memory/Hooks.cpp:55]: I was looking towards the end of the vtable because that's how I saw it in BDS.

Comment (brampedgex) [reply Memory/Hooks.cpp:55]: Yeah, it looks right to me. The code towards the end of the vtable in IDA was nearly identical to the lightEmission function in BDS. I'll double check this afternoon with a debugger when I'm back on my PC.

**Comment by GodGamer029** [comment]
Resolved

**Pull request closed without merging**